<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xml Editor</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTreeView" name="treeView">
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>150</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="Open"/>
    <addaction name="Save"/>
    <addaction name="Save_As"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="Undo"/>
    <addaction name="Redo"/>
    <addaction name="separator"/>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="Save_As">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="Undo">
   <property name="text">
    <string>Отменить действие</string>
   </property>
  </action>
  <action name="Redo">
   <property name="text">
    <string>Повторить действие</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Добавить элемент</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Удалить элемент</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
